### PR TITLE
Created an rfp page with contextual footers

### LINKED ISSUE #4082: RFP page - build
Build from copydoc: https://docs.google.com/document/d/1OA_Eg1200fHpt7YYkE4qubiWfxL_U5fnlA0OObRe2n8/edit#

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- created an RFP contact form with contextual footers
- cleaned up the copy doc
- didn't add it to the navigation.yaml for now
- updated the version of Django per Robin request
- looks like prettier cleaned up some sass

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/rfp](http://0.0.0.0:8001/rfp)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1OA_Eg1200fHpt7YYkE4qubiWfxL_U5fnlA0OObRe2n8/edit#heading=h.l28gk4yojgm6)
- Test the form
- See that it is linked from: http://0.0.0.0:8001/openstack/storage, http://0.0.0.0:8001/openstack/features and http://0.0.0.0:8001/openstack/consulting

## Issue / Card

Fixes #4082

### CODE REVIEW

**Comment by pmahnke** [comment]
@caldav and @Khutch25 and Thibaut - I think this page is a bit strange... and I am not sure about the url either

* you didn't write the contextual footer
* you didn't create a thank you page
* I had to ask for a form to be created
* the page just ends
* the messaging is a bit simplistic

however, I don't know the details of why it is needed, so I have built it, but I think it needs fixing.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Django==1.11.1
+Django==1.11.16
```
Comment (anthonydillon) [inline requirements.txt]: Next time we should bump deps in a separate PR as we may want to revert without losing work.

File: templates/rfp/index.html [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+  </div>
+</div>
+</section>
+
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-6">
+      <h2>We can help you with your Request for Information (RFI)</h2>
+      <p>If you are about to release an RFI,  RFP or RFQ -- please add Canonical, the company behind Ubuntu, to your list of vendors to solicit OpenStack and Kubernetes solutions. We can guide you to ask the right questions to find the optimal vendor solution for your specific needs.</p>
```
Comment (anthonydillon) [inline templates/rfp/index.html]: Please convert `--` to `&mdash;`

File: requirements.txt (hunk 1 above)
Comment (pmahnke) [reply requirements.txt]: I moved it to a new PR #4142

File: templates/rfp/index.html (hunk 2 above)
Comment (pmahnke) [reply templates/rfp/index.html]: Done

**Comment by kwm14** [comment]
LGTM 👍

**Comment by anthonydillon** [comment]
@pmahnke ready to go but you have conflicts

**Pull request merged**